addons/binding/org.openhab.binding.neeo/README.md

```diff
@@ -0,0 +1,161 @@
+NEEO Brains will be automatically discovered if MDNS/bonjour/zeroconf is installed on the local machine:
+
+1. On Windows - installing iTunes will install bonjour
+2. On Linux - please install zeroconf (see vendor documentation on how to do that)
+3. On Mac - should already be installed
+
+When you add the NEEO brain, the rooms on the brain will then be auto discovered and placed in the inbox. 
+
+If you have the brain both wired and wireless, the brain will NOT be discovered twice (only once) and which interface is discovered depends on the timing of the beacon discover message (first one wins).  If you discovered the wired first but want to use the wireless (or in the reverse), add the brain and then modify it's configuration to the IP address you want to use.
+
+If the brain is not discovered, here is list of the most common issues:
+
+1.  You didn't wait long enough.  I've noticed that it will take up to 5 minutes for the discovery to find the brain.
+2.  Local firewall is blocking the MDNS broadcast messages.  Modify the firewall to allow MDNS packets - typically port 5353 and/or IP address 224.0.0.251 
+3.  The brain is on a different subnet.  Unless you have special routing rules, having the brain on a different subnet than the openHAB instance will prevent discovery.  Either add routing rules or move one of them to the same subnet.
+4.  Bug in the MDNS library.  Occasionally a broadcast will be missed and a simple openHAB restart will fix the issue.
+5.  Brain isn't reachable.  Ping the brain's address from the openHAB machine and see if it responds.  
+
+If none of the above work, there are a few more things you can try
+
+1.  Use your local dns-sd command to see if you find the instance ("dns-sd -B _neeo" on windows 10)
+2.  Manually configure the brain and specify it's IP address
+3.  Look in the issues forum on the NEEO SDK Github - specifically the [Brain Discovery not working](https://github.com/NEEOInc/neeo-sdk/issues/36)
+
+
+## Thing Configuration
+
+The following are the configurations available to each of the bridges/things:
+
+### NEEO Brain
+
+| Name         | Type          | Description                               |
+|--------------|---------------|-------------------------------------------|
+| ipAddress    | string        | IP Address or host name of the NEEO Brain |
+
+### NEEO Room
+
+| Name         | Type          | Description                                           |
+|--------------|---------------|-------------------------------------------------------|
+| key          | string        | The unique key identifying the room on the NEEO brain |
+
+
+## Channels
+
+### NEEO Brain
+
+The NEEO brain has no channel information (although it has properties you may view within the paperUI).  The following properties are available at the time of this writing:
+
+| Name         | Description                                           |
+|--------------|---------------|-------------------------------------------------------|
```

Comment:
The column layout is not correct. Therefor it will not render as a table. If you click on View in github on the file you can see the effect. Also it's recommended to neatly align the bars | in the raw text (like the 2 tables above in this file).